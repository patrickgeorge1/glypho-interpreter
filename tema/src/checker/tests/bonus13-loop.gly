::zr+++R"""]%JJ%IIEIS?S?**_*^xx^^^p^NNA^DsD(KuKK<<jA|^4^>l>M";/=Ae^^0100GyGGGG[@%y6yg^gh4asfN|__<rBB2ooo