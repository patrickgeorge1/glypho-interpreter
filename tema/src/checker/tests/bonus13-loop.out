11101110010010
